src/modules/workspace/use.ts

```diff
@@ -26,6 +26,11 @@ export default async (name: string, options?) => {
   let production = options ? (options.p || options.production) : null
   let confirm
   const accountName = getAccount()
+
+  if(name === '-') {
+    name = getLastUsedWorkspace() || name
```

Comment:
I think it should throw if there's no `getLastUsedWorkspace`

Reply:
At least `.zsh` uses the current dir as last one when there is no history.

Reply:
I think it should throw because this will happen:

![image](https://user-images.githubusercontent.com/284515/60285813-ba797c00-98e4-11e9-924c-5be46d3454d3.png)

Reply:
I was using `git`s behaviour as a reference, which uses the actual `-` as the fallback. Anyway, totally agree about the `throw`

Reply:
I don't really like git's behaviour, if that's actually the case 😅
an error would be better IMO